nixpkgs/nixpkgs.bzl

```diff
@@ -70,6 +78,54 @@ nixpkgs_local_repository = repository_rule(
 def _is_supported_platform(repository_ctx):
     return repository_ctx.which("nix-build") != None
 
+def _expand_location(repository_ctx, string, labels, attr = None):
+    """Expand `$(location label)` to a path.
```

Comment:
To be consistent with Bazel's builtin location expansion this should error on unescaped instances of `$()` that are not `$(location ...)`.